### PR TITLE
simplify nested condition

### PR DESCRIPTION
**Pull request by cmucheru**
Simplify nested function for readability

### CODE REVIEW

File: src/backend/catalog/ag_catalog.c [hunk 1]
```diff
@@ -178,24 +177,19 @@ static void object_access(ObjectAccessType access, Oid class_id, Oid object_id,
      * The above applies to DROP TABLE command too.
      */
 
+    if (drop_arg->dropflags & PERFORM_DELETION_INTERNAL)
```
Comment (rafsun42) [inline src/backend/catalog/ag_catalog.c:180]: Does this change break for the case `class_id == RelationRelationId`?

**Comment by rafsun42** [comment]
Why are the files in `bc` extension?

**Comment by jrgemignani** [comment]
Why so many modified files?

File: src/backend/catalog/ag_catalog.c (hunk 1 above)
Comment (cmucheru) [reply src/backend/catalog/ag_catalog.c]: No. It does not break.

Comment (rafsun42) [reply src/backend/catalog/ag_catalog.c]: Could you elaborate more? In this case, the function calls `delete_label` first before returning. But with your change, it returns without calling that function. Does it have side effect?

**Comment by rafsun42** [comment]
@cmucheru The tests are failing at `catalog.sql`.

**Comment by cmucheru** [comment]
@rafsun42 thankyou. I am unfamiliar with the tests. I will like to check my code again to pass the tests in catalog.sql .

**Pull request closed without merging**